misc/diagram-util/src/test/resources/mainHttpCall/mainHttpCall.bal

```diff
@@ -0,0 +1,8 @@
+import ballerina/http;
+
+http:Client myclient = check new ("http://abcd");
```

Comment:
Is it okay to have standard library here? Does this import get resolved during the test phase?

Reply:
seems no hence the coverage issue... i will create my own client and use it ...